Swift-Base/AppDelegate.swift

```diff
@@ -29,6 +29,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         #endif
         
+        #if DEBUG
+            // print documents directory and device ID
+            print("\n*******************************************\nDOCUMENTS\n\(NSSearchPathForDirectoriesInDomains(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomainMask.UserDomainMask, true)[0])\n*******************************************\n")
```

Comment:
Are you mean all code or just added by me in this PR?
Maybe we should move all code from this method. For example setting up Crashlytics, MagicalRecord and etc